src/gh.js

```diff
@@ -31,24 +31,30 @@ async function getRegistrationToken() {
   }
 }
 
+// remove all runners with the input label
 async function removeRunner() {
```

Comment:
I considered that, but it would mean changing the modules interface (ie exports) and hence the users of it. Which is actually just the index.js I think. Went with the comment instead, maybe I'm being too picky.

Reply:
renames to removeRunners